validation_scripts/validation_workflow.smk

```diff
@@ -6,26 +6,26 @@ import shutil
 
```

Comment:
i think this is an old script that mathias used for validation that is not used at all... :) so maybe we can remove it

Reply:
That's okay by me. I have a few changes and general clean-ups that are unrelated to introducing shadow, but I think it's easiest to have a separate PR for that.